test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1606,3 +1606,312 @@ class instead of the class it was in when it was queued.
         self.server.deljob(jid3, wait=True)
 
         self.server.expect(JOB, {'job_state': 'R'}, id=jid5)
+
+    def test_multiple_job_preemption(self):
+        """
+        Test preemption of multiple jobs in same equivalence class
+        does not create a new equivalence class
+        """
+        # Create vnodes
+        a = {'resources_available.ncpus': 1}
+        self.server.create_vnodes('vnode', a, 1, self.mom, usenatvnode=True)
+
+        # Create expressq
+        a = {'queue_type': 'e', 'started': 't',
+             'enabled': 't', 'priority': 150}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='expressq')
+
+        # Submit 2 regular jobs
+        a = {'Resource_List.ncpus': 1}
+        J = Job(TEST_USER, attrs=a)
+        jid1 = self.server.submit(J)
+        jid2 = self.server.submit(J)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid1)
+        self.server.expect(JOB, {'job_state': 'Q'}, id=jid2)
+
+        # Suspend first job with qsig
+        self.server.sigjob(jobid=jid1, signal="suspend")
+        self.server.expect(JOB, {'job_state': 'S'}, id=jid1)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid2)
+
+        # Check equivalence classes =2, 1 for jid1 and one for jid2
+        self.scheduler.log_match("Number of job equivalence classes: 2",
+                                 max_attempts=10, starttime=self.t)
+
+        # Suspend job with a job from high priority queue
+        a = {'Resource_List.ncpus': 1, 'queue': 'expressq'}
+        Je = Job(TEST_USER, attrs=a)
+        jid3 = self.server.submit(Je)
+        self.server.expect(JOB, {'job_state': 'S'}, id=jid1)
+        self.server.expect(JOB, {'job_state': 'S'}, id=jid2)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid3)
+
+        """Only One equivalence class is present since both
+        suspended jobs are from the same equivalence class"""
+
+        self.scheduler.log_match("Number of job equivalence classes: 2",
+                                 max_attempts=10, starttime=self.t)
+
+    def test_multiple_job_preemtion_order(self):
+        """Test that when multiple jobs from same eqivalence class are
+        preempted in reverse order they were created they are placed
+        into the same equivalence class"""
```

Comment:
I combined test_multiple_job_preemption with this test and removed test_multiple_job_preemption